### PR TITLE
Add reputation_v2 and reputation_list to user author_profile

### PR DESCRIPTION
**Pull request by koutst**
- Add reputation and reputation list to user author profile

### CODE REVIEW

File: src/user/serializers.py [hunk 1]
```diff
@@ -708,10 +759,14 @@ def _group_and_serialize_actions(self):
...
-                    data["paper_title"] = (
+                    data[
+                        "paper_title"
+                    ] = (
                         thread_post.title
                     )  # paper_title instead of post_title for symmetry on the FE
-                    data["paper_id"] = (
+                    data[
+                        "paper_id"
+                    ] = (
```
Comment (gzurowski) [inline src/user/serializers.py:769]: Weird, our formatters are off. I remember this was just changed by one of my previous commits...

**Review by gzurowski — APPROVED** [review]
LGTM 👍

File: src/user/serializers.py (hunk 1 above)
Comment (koutst) [reply src/user/serializers.py:769]: My editor has it the original way but the pre commit changes it. Haven't looked into why.

Comment (gzurowski) [reply src/user/serializers.py:769]: Got it, we should take a look at that.

**Pull request merged**